pkg/storage/blockdev.go

```diff
@@ -190,3 +198,27 @@ func FilterEFISystemPartitions(devices []BlockDev) ([]BlockDev, error) {
 	}
 	return esps, nil
 }
+
+// GetMountpointByDevice gets the mountpoint by given
+// device name. Returns on first match
+func GetMountpointByDevice(devicePath string) (string, error) {
+	mountsPath, err := os.Open(LinuxMountsPath)
+	if err != nil {
+		return "", err
+	}
+
+	defer mountsPath.Close()
+	scanner := bufio.NewScanner(mountsPath)
+	scanner.Split(bufio.ScanLines)
```

Comment:
you don't need this call because `bufio.ScanLines` is already the default splitter

Reply:
Done